src/Container.js

```diff
@@ -0,0 +1,334 @@
+import { Bellhop } from 'bellhop-iframe';
+import { Features } from './Features';
+import { BasePlugin } from './plugins';
+
+//private/static variables
+const PLUGINS = [];
+
+/**
+ * The application container
+ * @class Container
+ * @property {Bellhop} client Communication layer between the container and application
+ * @property {Boolean} loaded Check to see if a application is loaded
+ * @property {Boolean} loading Check to see if a application is loading
+ * @property {Element} dom The DOM object for the iframe
+ * @property {HTMLElement} main The current iframe object
+ * @property {Object} release The current release data
+ * @property {object} options Optional parameteres
+ * @property {string} options.captionsButton selector for captions button
+ * @property {string} options.helpButton selector for help button
+ * @property {string} options.musicButton selector for music button
+ * @property {string} options.pauseButton selector for pause button
+ * @property {string} options.pauseFocusSelector The class to pause
+ * @property {string} options.sfxButton selector for sounf effects button
+ * @property {string} options.soundButton selector for captions button
+ * @property {string} options.voButton selector for vo button
+ * @static @property {Array} plugins The collection of Container plugins
+ * @static @property {String} version The current version of the library
+ *
+ * @constructor
+ * @param {string} iframeSelector selector for application iframe container
+ * @param {object} options Optional parameteres
+ * @param {string} options.helpButton selector for help button
+ * @param {string} options.captionsButton selector for captions button
+ * @param {string} options.soundButton selector for captions button
+ * @param {string} options.voButton selector for vo button
+ * @param {string} options.sfxButton selector for sounf effects button
+ * @param {string} options.musicButton selector for music button
+ * @param {string} options.pauseButton selector for pause button
+ * @param {string} options.pauseFocusSelector The class to pause
+ *        the application when focused on. This is useful for form elements which
+ *        require focus and play better with Application's keepFocus option.
+ */
+export class Container {
+  /**
+   *Creates an instance of Container.
+   * @param {*} iframeSelector
+   * @param {*} options
+   * @memberof Container
+   */
+  constructor(iframeSelector, options = {}) {
+    this.main = document.querySelector(iframeSelector);
+
+    if (null === this.main) {
+      throw new Error('No iframe was found with the provided selector');
+    }
+
+    this.client = new Bellhop();
+    this.dom = this.main;
+    this.loaded = false;
+    this.loading = false;
+    this.options = options;
+    this.release = null;
+    //Plugin init
+    this.plugins = PLUGINS.map(plugin => new plugin(this));
```

Comment:
I'm working on some Application plugin refactors. Before we merge, I'll want to compare notes and try to achieve parity.